apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -481,6 +483,13 @@ const EventTypePage = (props: EventTypeSetupProps) => {
               }
             }
             const { availability, ...rest } = input;
+
+            if (connectedCalendarsQuery.data?.connectedCalendars.length) {
+              defaultCalendar = connectedCalendarsQuery.data?.connectedCalendars
+                .map((connected) => connected.primary)
+                .find((cal) => cal?.externalId === input.destinationCalendar.externalId);
```

Comment:
When no destination calendar is selected and we use the default calendar `input.destinationCalendar` is set to null. We should fall back to the destination calendar email.